python/cuml/test/test_trustworthiness.py

```diff
@@ -24,10 +24,10 @@
 
 
 @pytest.mark.parametrize('input_type', ['ndarray', 'dataframe'])
-@pytest.mark.parametrize('n_samples', [10, 500])
-@pytest.mark.parametrize('batch_size', [512, 2])
+@pytest.mark.parametrize('n_samples', [150, 500])
 @pytest.mark.parametrize('n_features', [10, 100])
 @pytest.mark.parametrize('n_components', [2, 8])
+@pytest.mark.parametrize('batch_size', [128, 1024])
 def test_trustworthiness(input_type, n_samples, n_features, n_components,
```

Comment:
We should add a test for the case outlined in https://github.com/rapidsai/cuml/issues/3796 just to verify (as well as any other cases that you have enabled with this PR).